library/ajax/i18n_generator.php

```diff
@@ -12,7 +12,8 @@
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
-require_once(dirname(__FILE__) . "/../../interface/globals.php");
+$sessionAllowWrite = true;
```

Comment:
TODO
Gonna try to remove this (should be fine)

Reply:
This breaks tab name (undefined) if not there. Don't really understand why yet but will test now i've cleaned up main and removed jsFetchGlobals(a tear fell btw. I was proud of that!)

Reply:
huh, seems okay now! However, clinical reminders and encounters have slowed some after I pulled in your changes.
I'm looking over this.